pkg/build

```diff
@@ -5,29 +5,35 @@ srcdir ?= .
 AM_DEFAULT_VERBOSITY ?= 0
 NODE_ENV ?= production
 
+# Sorted by build time.  See webpack-benchmark rule below.
```

Comment:
The question stands - is there any measurable difference in build time now?
I am fine with this change of course, just wondering :)

Reply:
[benchmarking intensifies]

Reply:
Short answer, yes: about 9 seconds difference at `-j8` on my steady-thermal-state laptop (158s vs 149s).  At `-j4` it's 

Some insight into that: the biggest webpacks are `systemd` and `storaged`, and they both come almost at the end, which means that we very often get into a state where there are only two webpacks left running, with the other cores sitting idle.  Putting smaller webpacks at the end reduces the maximum latency we can hit as a result of this effect.

Because there are two smaller webpacks after `systemd` and `storaged`, I'd expect the benefit to substantially disappear at `-j2`, but I didn't test that.

Reply:
I've actually done some more careful benchmarking now, both before and after the changes from #17187 and in both cases, this is probably either a wash, or actually makes things slightly worse.  Let's not merge this.